cmd/shell/init.go

```diff
@@ -7,21 +7,22 @@ import (
 )
 
 const template = `#!/usr/bin/env zsh
-ANTIBODY_BINARY="%s"
+ANTIBODY_BINARY=$'%s'
```

Comment:
This line is merely a safety net as $'' is as quoted as we can get. The more important lines are below which fix quoting (`"$@"` is required over `$@` to get *exactly* repeatable arguments in all cases, even in ZSH.)

Using `command` ensures that any aliases or functions are never ran, avoiding any possibility of loops even in development.